src/snowflake/snowpark/functions.py

```diff
@@ -10775,6 +10800,48 @@ def model(
     )
 
 
+@publicapi
+def service(
+    service_name: str,
+    _emit_ast: bool = True,
+) -> Callable:
+    """
+    Creates a service function that can be used to call a service method.
+
+    Args:
+        service_name: The name of the service to call.
+
+    Example::
+
+        >>> original_role = session.get_current_role()
+        >>> original_db = session.get_current_database()
+        >>> original_schema = session.get_current_schema()
+        >>> try:
+        ...     session.use_role("test_role")
+        ...     session.use_database("tutorial_db")
+        ...     session.use_schema("data_schema")
+        ...     svc = service("tutorial_2_job_service")
+        ...     result_df = session.range(1).select(svc("SPCS_CANCEL_JOB")())
```

Comment:
You may be using this example for you easy test setup. But this is more useful for service with a function in it.

Make sure to have a public doc showcasing call via an argument to the function.